src/triedb.rs

```diff
@@ -19,7 +19,9 @@ use crate::node::Node as RlpNode;
 use crate::{Trie, TrieError};
 use ccrypto::{blake256, BLAKE_NULL_RLP};
 use cdb::HashDB;
+use lru_cache::LruCache;
 use primitives::H256;
+use std::cell::RefCell;
```

Comment:
I used RefCelll because of an error `'self' is a '&' reference, so the data it refers to cannot be borrowed as mutable`